regions/shapes/annulus.py

```diff
@@ -50,9 +54,23 @@ def inner_radius(self):
     def outer_radius(self):
         return self.region2.radius
 
+    @property
     def bounding_box(self):
         return self.region2.bounding_box()
 
+    @property
+    def area(self):
+        """Region area (float)."""
+        return self.region2.area() - self.region1.area()
```

Comment:
We probably need a check somewhere - is there one? - that region2 is larger than / encloses region 1

Reply:
there isn't any check..but the region1 is formed from inner_radius (argument) and region2 from outer_radius. I think there should be a check whether outer_radius is greater than inner_radius.

Reply:
one another thing though.when the region is inverted, the area kind of becomes undefined..we should check that. Isn't it??